File: test-project/greeter.provider.testfixture/src/test/java/testfixture/GreeterTest.java
package testfixture;

import examples.greeter.api.Greeter;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertFalse;

class GreeterTest extends GreeterFixture {
    @Test
    void testLocate() {
        Greeter greeter = locator().findGreeter();
        assertFalse(greeter.hello().isBlank());
    }
}

File: test-project-kotlin/greeter.provider.testfixture/src/test/kotlin/testfixture/GreeterTest.kt
package testfixture

import org.junit.jupiter.api.Test

import org.junit.jupiter.api.Assertions.assertFalse

class GreeterTest: GreeterFixture() {
    @Test
    fun testLocate() {
        val greeter = locator().findGreeter()
        assertFalse(greeter.hello().isBlank())
    }
}
